packages/app-store/googlecalendar/api/callback.ts

```diff
@@ -156,6 +158,23 @@ async function getHandler(req: NextApiRequest, res: NextApiResponse) {
   );
 }
 
+async function updateProfilePhoto(oAuth2Client: Auth.OAuth2Client, userId: number) {
+  try {
+    const oauth2 = google.oauth2({ version: "v2", auth: oAuth2Client });
+    const userDetails = await oauth2.userinfo.get();
+    if (userDetails.data?.picture) {
+      await prisma.user.updateMany({
```

Comment:
Using `updateMany` here since if the user already has a profile it would throw an error because no records were found to update the profile picture. I didn't want to flood our logs with error messages around this.

Reply:
A comment in code would be an good addition on the why we're doing this.